storage/innobase/srv/srv0srv.cc

```diff
@@ -3542,6 +3544,15 @@ srv_enable_redo_encryption()
 bool
 srv_enable_redo_encryption_mk()
 {
+	if (existing_redo_encryption_mode != REDO_LOG_ENCRYPT_MK
```

Comment:
Can we rewrite it as if( existing_redo_encryption_mode == REDO_LOG_ENCRYPT_RK)

Reply:
We can, but I don't think that would be better. This condition now exits on explicitly anything not handled by this function - that would return on one specific something not handled by it, which is more fragile if we ever add a third option.

Reply:
May be switch case then? that way new types are forced to handle in switch case